internal/cypress/sauce/runner.go

```diff
@@ -73,6 +77,36 @@ func (r *Runner) RunProject() (int, error) {
 	return exitCode, nil
 }
 
+// preliminarySteps do several checks before running Cypress tests.
+func (r *Runner) preliminarySteps() error {
```

Comment:
Would be good to give this method a more descriptive name, since it's unclear as to what steps this method should perform. Otherwise it has a chance to evolve into some large method over time because a lot of actions could be done here.